pkg/nodediscovery/nodediscovery.go

```diff
@@ -226,6 +226,8 @@ func (n *NodeDiscovery) UpdateCiliumNodeResource() {
 		return
 	}
 
+	log.WithField("node", nodeTypes.GetName()).Info("Creating or updating CiliumNode resource")
```

Comment:
We should use [this field from the current list of established fields](https://github.com/cilium/cilium/blob/0db8cd04465e88144a8f3b93c057053c2b07faa6/pkg/logging/logfields/logfields.go#L32), because this is an `Info` msg which will always be in the logs. I think the general pattern I've noticed is we add/use fields from the list of established fields when it's an `Info` msg or higher. `Debug` msgs don't need an entry in the established fields list.